client/index.js

```diff
@@ -283,14 +283,12 @@ addFilter(
 			accountStatus,
 			showUpdateDetailsTask,
 			wpcomReconnectUrl,
-			featureFlags: { accountOverviewTaskList },
 		} = wcpaySettings;
 
 		const wcPayTasks = getTasks( {
 			accountStatus,
```

Comment:
I think that now accountStatus should not be passed as it's missing in the interface https://github.com/Automattic/woocommerce-payments/pull/6696/files#diff-d773390b1a89b8ea2059380cd3f5c1997f373e3c6066cc06d3cec7299f50852fR24

Reply:
Good catch! Fixed in 50d919047.